service_manifest.yml

```diff
@@ -273,6 +277,21 @@ heuristics:
     name: PE file contains sections with non-standard names
     score: 0
 
+  # Inspired by https://github.com/viper-framework/viper-modules/blob/00ee6cd2b2ad4ed278279ca9e383e48bc23a2555/pe.py#L690
+  # and https://github.com/viper-framework/viper-modules/blob/00ee6cd2b2ad4ed278279ca9e383e48bc23a2555/lief.py#L1120
+  - description: No resources found.
+    filetype: "executable/windows"
+    heur_id: 35
+    name: PE file does not contain any resources
+    score: 100
+
+  # Inspired by https://github.com/viper-framework/viper-modules/blob/00ee6cd2b2ad4ed278279ca9e383e48bc23a2555/lief.py#L1243
+  - description: Lief could not find lang
+    filetype: "executable/windows"
+    heur_id: 36
+    name: No lang found
```

Comment:
Could we write languages instead of lang both in the name and description? :)

Reply:
https://github.com/CybercentreCanada/assemblyline-service-pe/pull/54/commits/d4902f05695a71bfc7ee9b86926a0cb0747af416